.env.example

```
@@ -54,7 +54,7 @@ database.tests.DBPrefix='ospos_'
 # ENCRYPTION
 #--------------------------------------------------------------------
 
-encryption.key=''
+encryption.key='2a873bf50976f6f0b174e614cd675bf71f02df252fa8fe57a05957f523375c22'
```

dotenv-linter (4.0.0)

[warning] [IncorrectDelimiter] The encryption.key key has incorrect delimiter

(IncorrectDelimiter)